packages/core/lib/commands/debug/meta.js

```diff
@@ -59,6 +63,11 @@ module.exports = {
         option: "--compile-none",
         description:
           "Forces the debugger to use artifacts even if it detects a problem.  Dangerous; may cause errors."
+      },
+      {
+        option: "--url",
+        description:
+          "Connects to a specified provider given via URL, ignoring networks in config. NOTE: compileNone is set to true when url is specified"
```

Comment:
I don't think we should be setting `compileNone` to `true` whenever `url` is specified.  I think we should only be doing it if `url` is specified and there's no config.

Reply:
fixed and updated description